File: Famoser.MassPass.Data/Entities/Communications/Request/Authorization/AuthorizationRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Famoser.MassPass.Data.Entities.Communications.Request.Base;

namespace Famoser.MassPass.Data.Entities.Communications.Request
{
    public class AuthorizationRequest : ApiRequest
    {
        public string AuthorisationCode { get; set; }

        public string UserName { get; set; }
        public string DeviceName { get; set; }
    }
}

File: Famoser.MassPass.Api/src/Models/Request/Authorization/AuthorizationRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 22/05/2016
 * Time: 22:46
 */

namespace Famoser\MassPass\Models\Request\Authorization;


use Famoser\MassPass\Models\Request\Base\ApiRequest;

class AuthorizationRequest extends ApiRequest
{
    public $AuthorisationCode;
    public $DeviceName;
}